execution/executor-test-helpers/src/integration_test_impl.rs

```diff
@@ -164,7 +173,7 @@ pub fn test_execution_with_storage_impl() -> Arc<AptosDB> {
 
     // Transfer 20k coins from account1 to account2.
     // balance: <1.98M, 1.22M, 1M
-    let txn4 = get_test_signed_transaction(
+    let txn4 = local_account1.sign_with_transaction_builder(
```

Comment:
I don't think this compiles,  you can run the test with `cargo x test -p executor`, this probably should be
```
let txn_factory = TransactionFactory::new(...);
let txn = local_account1.sign_with_transaction_builder(txn_factory.transfer(account2, 20_000));
```

Reply:
Got it!